src/main/java/io/wispforest/owo/util/VectorSerializer.java

```diff
@@ -135,4 +188,13 @@ public static Vector3f readf(PacketByteBuf buffer) {
         return new Vector3f(buffer.readFloat(), buffer.readFloat(), buffer.readFloat());
     }
 
+    /**
+     * Reads one vector from the given packet buffer
+     *
+     * @param buffer The buffer to read from
+     * @return The deserialized vector
+     */
+    public static Vector3f readi(PacketByteBuf buffer) {
+        return new Vector3f(buffer.readInt(), buffer.readInt(), buffer.readInt());
```

Comment:
Not the greatest idea to read integers from a buffer you've written floats to. Also like, don't write floats :poggus: